hbase-server/src/test/java/org/apache/hadoop/hbase/master/assignment/TestRegionMoveAndAbandon.java

```diff
@@ -102,6 +102,7 @@ public void test() throws Exception {
     LOG.info("Killing RS {}", rs1.getServerName());
     // Stop RS1
     cluster.killRegionServer(rs1.getServerName());
+    UTIL.waitFor(30_000, () -> rs1.isStopped() && !rs1.isAlive());
```

Comment:
Would it be better to increase the following to 60seconds? If let's say rs1 is stopped immediately, but 30 seconds is not enough for namespace table to be online. Increase the following to 60 seconds will make sure that enough time is waited.
 UTIL.waitTableAvailable(TableName.NAMESPACE_TABLE_NAME, 60_000);

Reply:
Let me commit w/. your suggestion @huaxiangsun